Traceback (most recent call last):
  File "_news_collector_ext.py", line 161, in <module>
    main()
  File "_news_collector_ext.py", line 141, in main
    cse_num = int(sys.argv[1])
IndexError: list index out of range
mv: cannot stat ‘all_*’: No such file or directory
